### ISSUE TITLE
 Invalid format '%d'; use format %s for character objects

### ISSUE THREAD

**Issue by chipoglesby** [artemklevtsov/RGA#2]
Artem,

I get this response when trying to query data with more than 10,000 rows:

``` r
Response contain more then 10000 rows. Batch processing mode enabled.
Fetching page 2 of 3...
Fetching page 3 of 3...
Obtained data.frame with 20813 rows and 3 columns.
Error in sprintf("Data contains sampled data. Used %d sessions (%1.0f%% of sessions).",  : 
invalid format '%d'; use format %s for character objects
```

**Comment by artemklevtsov** [comment]
Can you try the most latest dev version? Also it would be helpful if you provide a reproducible code example.

**Comment by artemklevtsov** [comment]
This error was fixed in c4fb53e.
